dind/docker-iptables-glue/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/inloco/docker-iptables-glue
+
+go 1.14
```

Comment:
I should be using Go 1.15 instead, it is the latest version and it is also the one used by the operator.

Reply:
a2643b2